lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/nested-journey-definitions.yaml

```diff
@@ -26,4 +26,4 @@ ADDRESS_AND_FRAUD:
         next:
           exitEventToEmit: next
         enhanced-verification:
-          exitEventToEmit: next
+          exitEventToEmit: fraudEnhanced
```

Comment:
Can we just use `enhanced-verification` for this - is there a reason you think we need a new name?

Reply:
I thought, we can follow with separate exitEvent for Fraud. Yeah, you are right. it's not necessary. changed and deployed to my dev env and tested again.